src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/EncoderExtensions.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.IO;
 
-namespace System.Text.Encodings.Web
+namespace Microsoft.Framework.WebEncoders
```

Comment:
The plan used to be that we will leave the interfaces as is and implement these on adapters. But that meant that we would have 6 almost unnecessary types: 3 interfaces and 3 adapters. 
Yesterday, we decided to pursue the option of adding 3 abstractions (HtmlEncoder, UrlEncoder, etc). After this change, there is no need for these interfaces and adapters, and so I moved them to a "to be deleted" namespace.

Normally, I would just remove them instead of moving to a different namespace, but ASP.NET code depends on these and so I want to first transition ASP.NET to the new abstractions and then do the removal.